src/vue/backend/src/main.js

```diff
@@ -7,9 +7,10 @@ import IDXStrapClass from '@idxbrokerllc/idxstrap/dist/idxStrap.js'
 
 // Import VCL base and variable styles
 import '@idxbrokerllc/idxstrap/dist/styles/base.scss'
+import '@idxbrokerllc/idxstrap/dist/styles/globalVariables.scss'
```

Comment:
You should not need `globalVariables.scss` because it's loaded in globally in the vue config file: https://github.com/idxbroker/wordpress-plugin/blob/release/3.0.0/src/vue/backend/vue.config.js#L33